test/integration/smoke/test_activate_template.py

```diff
@@ -98,10 +105,6 @@ def setUp(self):
         self.activateTemplateCmd = activateSystemVMTemplate.activateSystemVMTemplateCmd()
         self.listConfigurationsCmd = listConfigurations.listConfigurationsCmd()
         self.listConfigurationsCmd.name = 'router.template.kvm'
```

Comment:
please do not hardcode, this feature is not just kvm specific

Reply:
Done